README.md

```diff
@@ -100,11 +100,79 @@ this can be used with `watch`.
 watch -c juju status --color
 ```
 
-### Alternate deployment methods
+## Alternate deployment methods
 
+### Deploying storage
 
+The Canonical Distribution of Kubernetes allows you to connect with durable
+storage devices such as [Ceph](http://ceph.com). When paired with the
+[Juju Storage](https://jujucharms.com/docs/2.0/charms-storage) feature you
+can add durable storage easily and across all clouds.
 
-#### Usage with your own binaries
+Deploy the ceph-mon and ceph-osd charms.
+
+```
+juju deploy cs:ceph-mon -n 3
+juju deploy cs:ceph-osd -n 3
```

Comment:
AIUI - both applications need to be deployed as a multiple of three to satisfy the config of replicas. I don't think this is a tuneable value, and we have to satisfy that minimum requirement in order for ceph to quorem.